app/lib/network_utils.py

```diff
@@ -682,3 +683,33 @@ async def set_collator_nodes_keys_on_chain(para_id, nodes=[], statefulset=''):
     for node_name in nodes:
         ss58_format = get_substrate_node(node_name).get("ss58_format")
         collator_set_keys(node_name, para_id, ss58_format)
+
+
+def get_substrate_runtime(node_client):
+    last_runtime_upgrade = node_client.query("System", "LastRuntimeUpgrade").value
+    wasm = get_parachain_wasm(node_client)
```

Comment:
I think the function `get_parachain_wasm` can be renamed `get_chain_wasm` as it is actually agnostic of what kind of chain it is.